spec/serializers/api/v3/finance/statement_serializer_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Api
+  module V3
+    module Finance
+      RSpec.describe StatementSerializer do
+        describe "serialization" do
+          let(:cohort) { create(:cohort, :next) }
+          let(:statement) { create(:ecf_statement, name: "January 2022", cohort:) }
+          let(:subject) { described_class.new(statement).serializable_hash }
```

Comment:
I'd decompose that differently:

```ruby
subject(:serialiser) { described_class.new(statement) }

 it "returns the month" do
  expect(serialiser.serializable_hash[:data][:attributes][:month]).to eq("January")
end
```
I think it's a bit less confusing to have the subject be the instance of what is in between `RSpec.describe` _the_thing_we_want_to_test_the_behaviour_of_ `do`, which is the default behaviour. 
And in the body of the test be explicit about what method / behaviour we are testing. 
This way the `subject` never changes and you don't have to jump around the specs to keep track of what the subject is.